doc/source/index.rst.template

```diff
@@ -146,6 +146,7 @@ See the package overview for more detail about what's in the library.
     comparison_with_r
     comparison_with_sql
     comparison_with_sas
+    performance
```

Comment:
You can start a new section at the same level as `Cython (Writing C extensions for pandas)` though. And maybe even put it first, since people should be fixing these issues before moving to cython.